services/app-api/forms/mcpar.json

```diff
@@ -3420,7 +3488,7 @@
               "spreadsheet": "D4_Plan_ILOS",
               "info": [
                 {
-                  "type": "span",
+                  "type": "p",
```

Comment:
Same here? I may not be understand the structure breakdown for when the `children` prop needs to be passed.